.github/workflows/scripts/check-creators-workflows-code.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -eo pipefail
+
+SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+BASE_DIR="$(
+  cd $SCRIPT_DIR && cd ../.. &
+  pwd
+)"
+
+CREATOR_WORKFLOWS_DIR=$BASE_DIR/creators/extension/app/files/.github/workflows
+
+WORKFLOW_BRANCH=$1
+
+PROPER_WORKFLOW_CHARTS="name: Build and Release Extension Charts
+
+on:
+  workflow_dispatch:
+  release:  
+    types: [released]
+
+defaults:
+  run:
+    shell: bash
+    working-directory: ./
+
+jobs:
+  build-extension-charts:
+    uses: rancher/dashboard/.github/workflows/build-extension-charts.yml@$WORKFLOW_BRANCH
+    permissions:
+      actions: write
+      contents: write
+      deployments: write
+      pages: write
+    with:
+      target_branch: gh-pages
+      tagged_release: \${{ github.ref_name }}"
+
+PROPER_WORKFLOW_CATALOG="name: Build and Release Extension Catalog
+
+on:
+  workflow_dispatch:
+  release:
+    types: [released]
+
+defaults:
+  run:
+    shell: bash
+    working-directory: ./
+
+jobs:
+  build-extension-catalog:
+    uses: rancher/dashboard/.github/workflows/build-extension-catalog.yml@$WORKFLOW_BRANCH
+    permissions:
+      actions: write
+      contents: read
+      packages: write
+    with:
+      registry_target: ghcr.io
+      registry_user: \${{ github.actor }}
+      tagged_release: \${{ github.ref_name }}
+    secrets: 
+      registry_token: \${{ secrets.GITHUB_TOKEN }}"
```

Comment:
Wouldn't we need to update this script every time we update those workflow files? There should be an easier way to just pull the workflow files directly from the git repo.

I might be misunderstanding the point of this PR, but it seems to be superfluous to just ensure our workflow matches a set scaffold.

Reply:
Given this insight, I may recommend you use GH Actions for this purpose, so they can just import it and bump the version: https://docs.github.com/en/actions/sharing-automations/creating-actions/about-custom-actions

Not sure if we also need to publish it, but I guess that would not be bad. The principle seems similar to the template by using release tags: https://docs.github.com/en/actions/sharing-automations/creating-actions/publishing-actions-in-github-marketplace

Reply:
Alex and I spoke about this, I think the way he has implemented should be fine for the time being.